web/src/main/java/org/akaza/openclinica/controller/openrosa/OpenRosaSubmissionController.java

```diff
@@ -251,7 +251,9 @@
 
         int count = 0;
         for (EventCrf evCrf : eventCrfs) {
-            if (evCrf.getStatusId() == org.akaza.openclinica.domain.Status.UNAVAILABLE.getCode()) {
+            if (evCrf.getStatusId() == org.akaza.openclinica.domain.Status.UNAVAILABLE.getCode()
```

Comment:
There are 2 different imports for Status class in OC.
 org.akaza.openclinica.bean.core.Status;
 org.akaza.openclinica.domain.Status
The first one has already been imported and in use in another method and I need to use
2nd option with hibernate.